src/features/application/components/Apply.js

```diff
@@ -124,19 +192,80 @@ function Apply() {
             <FormControl label="Job Position">
               <Select
                 name="jobPosition"
-                defaultValue={options[0]}
-                options={options}
+                defaultValue={jobPositions[0]}
```

Comment:
- Try to use defensive programming as much as possible, so that the app doesn't crash. So in this case, a check is there an array and first element inside the array: `jobPositions && jobPositions.length ? jobPositions[0] : {}`